### PR TITLE
Add Backblaze B2Sync Latest

### PR DESCRIPTION
**Pull request by BenjaminHCCarr**
#### Adding a new cask
- [X] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [X] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [X] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [X] Commit message includes cask’s name.
- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` left no offenses.
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.

Add support for Backblaze low-cost backup storage syncing application.
https://www.backblaze.com/b2/docs/b2sync.html

### CODE REVIEW

File: Casks/b2sync.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'b2sync' do
+  version ':latest'
+  sha256 '93b11c9ea15077bc894ac0634dd999022d1e8c7ad1ddcdff3e78f9ea3fa27f85'
+
+  url 'https://www.backblaze.com/b2/docs/b2sync_for_mac.zip'
+  name 'B2Sync for Mac'
+  homepage 'https://www.backblaze.com/b2/docs/b2sync.html'
+  license :mit
+
+  app 'b2sync_for_mac/b2sync'
```
Comment (vitorgalvao) [inline Casks/b2sync.rb]: Is this a directory?

File: Casks/b2sync.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'b2sync' do
+  version ':latest'
```
Comment (vitorgalvao) [inline Casks/b2sync.rb]: Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/f25b6babcd398abf48e33af3d887b2d00de1d661/Casks/kibako.rb).

File: Casks/b2sync.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  sha256 '93b11c9ea15077bc894ac0634dd999022d1e8c7ad1ddcdff3e78f9ea3fa27f85'
+
+  url 'https://www.backblaze.com/b2/docs/b2sync_for_mac.zip'
+  name 'B2Sync for Mac'
```
Comment (vitorgalvao) [inline Casks/b2sync.rb]: ` for Mac` should be removed.

**Comment by BenjaminHCCarr** [comment]
Yes, it is a directory with a binary.
I will update :latest and sha checks and remove for mac.

I yanked it because I wasn't expecting the string of commits left over from securitygrowler that you and I did two weeks ago. I thought I had squashed them already, but I must have pulled again before pushing, I will try to make the edits and rebase tonight. baby woke up.

File: Casks/b2sync.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+cask 'b2sync' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://www.backblaze.com/b2/docs/b2sync_for_mac.zip'
+  name 'B2Sync'
+  homepage 'https://www.backblaze.com/b2/docs/b2sync.html'
+  license :mit
+
+  app 'b2sync_for_mac/b2sync'
```
Comment (vitorgalvao) [inline Casks/b2sync.rb]: This should be `binary`, not `app`, since it’s a CLI tool.

**Comment by vitorgalvao** [comment]
> I will try to make the edits and rebase tonight. baby woke up.

Please, no rush. Human beings (especially defenseless ones) first.

Why this tool, though? [On the page](https://www.backblaze.com/b2/docs/b2sync.html) they seem to suggest not using it, but using [a different tool instead](https://www.backblaze.com/b2/docs/quick_command_line.html).

**Comment by BenjaminHCCarr** [comment]
This all becomes moot when JCLOUDS as B2 which is getting closer:

https://issues.apache.org/jira/browse/JCLOUDS-1005

As I use rsync-over-ssh for multiple backups, this looked handy. But you are probably correct, they do describe the package as alpha. And I should not be lazy and just write a b2 script to do this. The idea of a pre-rolled rsync to a bucket that miniimized transaction counts sounded nice.

the B2 package is already rolled at: https://github.com/Homebrew/homebrew-core/blob/master/Formula/b2-tools.rb

**Pull request closed without merging**